nx_parallel/partition.py

```diff
@@ -0,0 +1,136 @@
+import inspect
+import joblib
+import itertools
+import networkx as nx
+from functools import reduce
+from typing import Optional, Iterable, Callable, Tuple, Union, List, Dict, Any
+from nx_parallel.external import Backend
+
+class NxMap:
+    """Map a function to an iterable, and call distributed execution."""
+
+    def __init__(self, func: Callable, iterator: str, backend: str = "multiprocessing", processes: Optional[int] = None):
+        self.func = func
+        self.backend = Backend(backend, processes)
+        self.iterator = iterator
+
+    @staticmethod
+    def chunk(l: Union[List, Tuple], n: int) -> Iterable:
+        """Divide a list `l` of nodes or edges into `n` chunks."""
+        l_c = iter(l)
+        while 1:
+            x = tuple(itertools.islice(l_c, n))
+            if not x:
+                return
+            yield x
+
+    def create_iterables(self, G: nx.Graph, iterator: str) -> Iterable:
```

Comment:
These 2 function have been included in the nx-parallel. Thanks @dPys !

Reply:
Awesome